ci_framework/roles/set_openstack_containers/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: 'Make sure {{ cifmw_set_openstack_containers_basedir }} exists'
+  ansible.builtin.file:
+    path: "{{ cifmw_set_openstack_containers_basedir }}/artifacts"
+    state: directory
+
+- name: Set container tag from dlrn md5 hash file
+  when:
+    - cifmw_set_openstack_containers_tag_from_md5 | bool
+  block:
+    - name: Get the md5 hash from hash file
+      ansible.builtin.slurp:
+        src: "{{ cifmw_set_openstack_containers_dlrn_md5_path }}"
+      register: dlrn_md5
+
+    - name: Set container tag
+      ansible.builtin.set_fact:
+        cifmw_set_openstack_containers_tag: "{{ dlrn_md5['content'] | b64decode | trim }}"
+        cacheable: true
+
+- name: Set the operator name
+  ansible.builtin.set_fact:
+    operator_name: "{{ cifmw_set_openstack_containers_operator_name }}"
+    cacheable: true
+
+- name: Get all openstack pods
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell:
+    cmd: oc get pods -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o name
+  register: operator_pod
+
+- name: list all pods
+  ansible.builtin.debug:
+    msg: "{{ operator_pod }}"
```

Comment:
if you want to keep the debug, please use `var: operator_pod` instead of the "msg". It would show the content in a cleaner way.